roles/ovirt.hosts/tasks/main.yml

```diff
@@ -2,9 +2,9 @@
 - name: Get hosts
   ovirt_host_info:
     auth: "{{ ovirt_auth }}"
-    pattern: "{{ hosts | map(attribute='name') | map('regex_replace', '(.*)', 'name=\\1') | list | join(' or ') }} status=installfailed"
+    pattern: "{{ ovirt_infra_hosts | map(attribute='name') | map('regex_replace', '(.*)', 'name=\\1') | list | join(' or ') }} status=installfailed"
   register: host_info
-  when: "hosts is defined"
+  when: ovirt_infra_hosts | bool
```

Comment:
would use if `ovirt_infra_hosts` or `ovirt_infra_hosts is not []` because the `bool` filter only converts string `"True"` to bool

Reply:
I implemented the comment but I still it would be great if we could get somebody else than me to test it before merging